modules/openy_features/openy_node/modules/openy_node_category/config/install/core.entity_form_display.node.program_subcategory.default.yml

```diff
@@ -14,6 +14,7 @@ dependencies:
     - field.field.node.program_subcategory.field_sidebar_content
     - node.type.program_subcategory
   module:
+    - content_moderation
```

Comment:
I guess we should be pretty confident about moderation enabled for specific content types.
Let's discuss this on our upcoming call, but I'd rather remove moderation from all CTs other than one most used ( like Landing Page )